Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/General/OutlookGridAlphabeticGroup.cs

```diff
@@ -72,9 +66,13 @@ public override object Value
             {
                 //Note : value with Clone() is already 1 character, but no problem here
                 if (value != null && !string.IsNullOrEmpty(value.ToString())) //useful for textand image object
+                {
                     base.Value = value.ToString().Substring(0, 1).ToUpper();
+                }
                 else
+                {
                     base.Value = "";
```

Comment:
Should state `string.Empty` so this does not look like some forgotten string.

Reply:
Changed